dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java

```diff
@@ -222,11 +222,11 @@ public static void transformApp(Path appPath, Path newAppPath) {
         final String m = "transformApp";
         Log.info(c, m, "Transforming app: " + appPath);
 
-        // Setup FileOutputStream and never use.  JakartaTransformer defaults cause OOM
+        // Setup file output stream and only keep if we fail
         FileOutputStream fos = null;
-        File throwAwayFile = new File("transformer_output.log");
+        File outputLog = new File("results/transformer_output.log");
```

Comment:
I wonder instead of just being called transformer_output.log having a name that includes the app name could be helpful if multiple apps fail for the same test suite.